### PR TITLE
lmms 1.2.0: add initial recipe and patchset

### PR DESCRIPTION
**Pull request by kenmays**
NOTE: Fixed patchset so it applies cleanly to lmms 1.2.0 stable release.
Recipe disabled for full build review.
Lots of bugs fixes from RC8 build.

Built on Haiku R1B1 hrev 52295+129 w/ Qt 5.12.2-1.

### CODE REVIEW

File: media-sound/lmms/lmms-1.2.0.recipe [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+SOURCE_DIR_2="rpmalloc-$srcGitRev2"
+#PATCHES="lmms-$portVersion.patchset"
+ADDITIONAL_FILES="LMMS.rdef"
+
+ARCHITECTURES="!x86_gcc2 !x86_64"
+SECONDARY_ARCHITECTURES="!x86"
+
+PROVIDES="
+	lmms$secondaryArchSuffix = $portVersion
+	app:lmms$secondaryArchSuffix
```
Comment (Begasus) [inline media-sound/lmms/lmms-1.2.0.recipe]: Didn't check the build, but "app:lmms" (no need for secondaryArchSuffix normally) should use the same as the actual app "LMMS"?

**Comment by korli** [comment]
Please remove old recipe and patchset.

File: media-sound/lmms/lmms-1.2.0.recipe [hunk 2]
```diff
@@ -10,17 +10,17 @@ COPYRIGHT="2004-2019 Tobias Doerffel et al."
...
 SOURCE_URI="https://github.com/LMMS/lmms/archive/v${portVersion/\~/-}.tar.gz"
-CHECKSUM_SHA256="4511687e47f6159cb2a250528bcccaa38b36b78f345466d090514214888943e8"
+CHECKSUM_SHA256="0c1be741b39b20023a34f16bad043ad668d054a0d776cc1e083d80f9cc9cb6a7"
 SOURCE_DIR="lmms-${portVersion/\~/-}"
 srcGitRev2="a66360f09e91cca9c7a6a16162bff6c3bd1f7efb"
 SOURCE_URI_2="https://github.com/rampantpixels/rpmalloc/archive/$srcGitRev2.tar.gz"
 CHECKSUM_SHA256_2="1aa23b2395eacc0fd86f197f0c4dffeb69107f5e0b73c2eed5c6b86ed2bef2b7"
 SOURCE_DIR_2="rpmalloc-$srcGitRev2"
-PATCHES="lmms-$portVersion.patchset"
+#PATCHES="lmms-$portVersion.patchset"
```
Comment (korli) [inline media-sound/lmms/lmms-1.2.0.recipe]: Looks wrong.

**Comment by korli** [comment]
If the build is broken for all architectures, let's just drop the PR.

File: media-sound/lmms/lmms-1.2.0.recipe [hunk 3]
```diff
@@ -48,7 +48,7 @@ REQUIRES="
 BUILD_REQUIRES="
 	haiku${secondaryArchSuffix}_devel
 	devel:libfftw3f$secondaryArchSuffix
-	devel:libfluidsynth$secondaryArchSuffix >= 2
+	devel:libfluidsynth$secondaryArchSuffix
```
Comment (korli) [inline media-sound/lmms/lmms-1.2.0.recipe]: Why remove the version requirement?

Comment (kenmays) [reply media-sound/lmms/lmms-1.2.0.recipe]: Breaks hp pkg downloads. Is there a setting to override?

Comment (diversys) [reply media-sound/lmms/lmms-1.2.0.recipe]: > Breaks hp pkg downloads. Is there a setting to override?

This should have been fixed in https://github.com/haikuports/haikuporter/commit/2cc27775286313a45363dde360e0e6385adfa614.

Comment (korli) [reply media-sound/lmms/lmms-1.2.0.recipe]: Please restore the version requirement

**Comment by kenmays** [comment]
Can close this one. Recent Qt 5.13 update breaks build. Will wait upstream testing with newer Qt releases.

**Pull request closed without merging**